apps/api/v2/src/modules/organizations/repositories/organizationUsers.repository.ts

```diff
@@ -0,0 +1,29 @@
+import { PrismaReadService } from "@/modules/prisma/prisma-read.service";
```

Comment:
File casing - we use casing like this - `organization-users.repository.ts`.

The same goes for other files in this PR aka `listUsersResponse.dto.ts`, `isemailStringOrArray.ts`.